### PR TITLE
Dont overwrite existing file on create

### PR DESCRIPTION
**Pull request by samlhuillier**

### CODE REVIEW

**Comment by chuyouchia** [comment]
But we still rewrite the whole file, when we are modifying the file regardless of whether something changes or not right? That can mess with the temporal agent's understanding of whether a note has been modified recently or not

**Review by chuyouchia — APPROVED** [review]

File: electron/main/Files/Filesystem.ts [hunk 1]
```diff
@@ -116,10 +116,15 @@ export function createFileRecursive(
...
+
   if (!fs.existsSync(dirname)) {
     fs.mkdirSync(dirname, { recursive: true });
   }
 
+  if (fs.existsSync(filePath)) {
+    return;
+  }
+
   fs.writeFileSync(filePath, content, charset);
```
Comment (chuyouchia) [inline electron/main/Files/Filesystem.ts:128]: This line looks like dead code now that the prior 2 covers all possible scenarios

**Review by chuyouchia — CHANGES_REQUESTED** [review]

File: electron/main/Files/Filesystem.ts (hunk 1 above)
Comment (samlhuillier) [reply electron/main/Files/Filesystem.ts:128]: Wdym?

Comment (chuyouchia) [reply electron/main/Files/Filesystem.ts:128]: sorry I misread it...I thought the existsSync was being applied to the same `dirname`

**Review by chuyouchia — APPROVED** [review]

**Pull request merged**